Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -500,12 +500,9 @@ open class NavigationMapView: UIView {
             guard let layers = try? mapView.__map.getStyleLayers().reversed() else { return nil }
             for layer in layers {
                 if !(layer.type == "symbol") && !identifiers.contains(layer.id) {
-                    let source = try? mapView.__map.getStyleLayerProperty(forLayerId: layer.id, property: "source").value as? String
```

Comment:
This might have been influenced by how, in v1._x_, we check whether the layer is backed by the Mapbox Streets source, to avoid getting thrown off by developer-defined vector tile sources that happen to have similar source layer names. But as long as the source layer names are reasonably descriptive, it shouldn’t be a major problem to omit the source check.